File: avalon-modules/avalon-dataset/avalon-dataset-repository/src/main/java/io/github/lishangbu/avalon/dataset/repository/PokemonStatRepository.java
package io.github.lishangbu.avalon.dataset.repository;

import io.github.lishangbu.avalon.dataset.entity.PokemonStat;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

/// 宝可梦能力值数据访问接口
///
/// 提供宝可梦能力值数据的CRUD操作
///
/// @author lishangbu
/// @since 2026/2/16
@Repository
public interface PokemonStatRepository
        extends JpaRepository<PokemonStat, PokemonStat.PokemonStatId> {}

File: avalon-modules/avalon-dataset/avalon-dataset-repository/src/main/kotlin/io/github/lishangbu/avalon/dataset/repository/PokemonStatRepository.kt
package io.github.lishangbu.avalon.dataset.repository

import io.github.lishangbu.avalon.dataset.entity.PokemonStat
import io.github.lishangbu.avalon.dataset.entity.PokemonStatId
import org.babyfish.jimmer.spring.repository.KRepository
import org.springframework.stereotype.Repository

/**
 * 宝可梦能力值仓储接口
 *
 * 定义宝可梦能力值数据的查询与持久化操作
 *
 * @author lishangbu
 * @since 2026/2/16
 */
@Repository
interface PokemonStatRepository : KRepository<PokemonStat, PokemonStatId>
